docs/scripts.js

```diff
@@ -1,31 +1,35 @@
-function slideExampleCode(e) {
-  e.preventDefault();
+function slideExampleCode(event) {
+  event.preventDefault();
 
-  var slide = e.target.parentElement.nextElementSibling;
+  var slide = event.target.parentElement.nextElementSibling;
   slide.classList.toggle('open');
 
-  if(slide.classList.contains('open')) {
-    slide.style.maxHeight = slide.scrollHeight + 'px'; // open the example code
+  if (slide.classList.contains('open')) {
+    // open the example code
+    slide.style.maxHeight = slide.scrollHeight + 'px';
   } else {
-    slide.style.maxHeight = 0; // close the example code
+    // close the example code
+    slide.style.maxHeight = 0;
   }
 }
 
 document.addEventListener('DOMContentLoaded', function() {
-  var toggleButtons = document.getElementsByClassName('example-code-link'); // get the buttons
+  // get the buttons
+  var toggleButtons = document.querySelectorAll('.example-code-link');
 
-  for(var i = 0; i < toggleButtons.length; i++) {
-    toggleButtons[i].addEventListener("click", slideExampleCode); // add the event to each button
+  for (var i = 0; i < toggleButtons.length; i++) {
```

Comment:
I could probably use one variable, but I went with the way I'd go if this was a let variable. Feel free to change it later :)